packages/build/src/utils/blobs.ts

```diff
@@ -12,6 +14,40 @@ export const getBlobsDirs = (buildDir: string, packagePath?: string) => [
   path.resolve(buildDir, packagePath || '', LEGACY_BLOBS_PATH),
 ]
 
+interface EnvironmentContext {
+  api?: {
+    host: string
+    scheme: string
+  }
+  deployId?: string
+  siteId?: string
+  token?: string
+}
+
+// TODO: Move this work to a method exported by `@netlify/blobs`.
```

Comment:
Something I've done in the past is attach a ticket number to TODO comments. This helps ensures that the work gets finished, gives a space to provide more detailed context, and makes it easier for other people to pick up the work.